config/azure_config.go

```diff
@@ -0,0 +1,33 @@
+package config
+
+import "os"
+
+type AzureConfig struct {
+	AzureStorageAccount        string
+	AzureStorageAccessKey      string
+	AzureStorageContainerName  string
+	AzureStorageOpenBankSkPath string
+	AzureStorageOpenBankSkName string
+	AzureTenantId              string
+	AzureClientId              string
+	AzureClientSecret          string
+	AzureStorageUploadPath     string
+	AzureStorageFallbackFolder string
+	VaultName                  string
+}
+
+func getAzureConfig() AzureConfig {
```

Comment:
Opa, sem voltar o ponteiro, tu vai ter criado um objeto(struct), e voltar uma cópia dele 😀 
```suggestion
func getAzureConfig() *AzureConfig {
```